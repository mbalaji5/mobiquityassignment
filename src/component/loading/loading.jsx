import React from "react";
// Loading component.
const Loading = () => {
  return <div>...Loading</div>;
};
export default Loading;
