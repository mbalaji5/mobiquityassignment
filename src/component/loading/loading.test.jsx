import { render, screen } from "@testing-library/react";
import { Loading } from "../loading";
describe("should validate loading component", () => {
  it("should validate loading componet text element", () => {
    render(<Loading />);
    const element = screen.getByText(/..Loading/i);
    expect(element).toBeDefined();
  });
});
describe("Should validate loading", () => {
  it("Should validate shopshot", () => {
    const { baseElement } = render(<Loading />);
    expect(baseElement).toMatchSnapshot();
  });
});
