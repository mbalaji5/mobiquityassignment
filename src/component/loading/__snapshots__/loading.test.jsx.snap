// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should validate Loading Should validate shopshot 1`] = `
<body>
  <div>
    <div>
      ...Loading
    </div>
  </div>
</body>
`;

exports[`Should validate loading Should validate shopshot 1`] = `
<body>
  <div>
    <div>
      ...Loading
    </div>
  </div>
</body>
`;
